app/src/main/java/com/duckduckgo/app/browser/httperrors/HttpErrorPixels.kt

```diff
@@ -72,8 +169,38 @@ class RealHttpErrorPixels @Inject constructor(
         return "${this.pixelName}_count"
     }
 
+    private fun isVpnConnected(
+        netpEntitlementStatus: Boolean,
+        connectionState: ConnectionState,
+        subscriptionStatus: SubscriptionStatus,
+    ): Boolean {
+        if (subscriptionStatus in setOf(
+                SubscriptionStatus.AUTO_RENEWABLE,
+                SubscriptionStatus.NOT_AUTO_RENEWABLE,
+                SubscriptionStatus.GRACE_PERIOD,
+            )
+        ) {
+            if (netpEntitlementStatus) {
+                return connectionState.isConnected()
+            }
+        }
+        return false
+    }
+
+    private suspend fun isExternalVpnDetected(pProVpnConnected: Boolean): Boolean {
+        if (pProVpnConnected) return false
```

Comment:
You can use `ExternalVpnDetector.isExternalVpnDetected` which is under `vpn-api` module. You don’t need to check `pProVpnConnected` nor `vpnFeaturesRegistry.isAnyFeatureRunning()` as it is already taken care of there.